portal-backend/loader/dataset_loader/single_input_file_dependency_loader.py

```diff
@@ -50,9 +50,6 @@ def compound_dose_lookup(x):
     if DependencyEnum.is_compound_experiment_enum(dataset_enum):
         entity_lookup = compound_experiment_lookup
         entity_type = "compound_experiment"
-    elif DependencyDataset.is_dose_enum(dataset_enum):
-        entity_lookup = compound_dose_lookup
```

Comment:
I think this is the only place compound_dose_lookup is used, so you could probably delete that, too!